apps/manager_app/manager_api.h

```diff
@@ -0,0 +1,60 @@
+/**
+ * @file    manager_api.h
+ * @author  Cypherock X1 Team
+ * @copyright Copyright (c) 2023 HODL TECH PTE LTD
+ * <br/> You may obtain a copy of license at <a href="https://mitcc.org/"
+ * target=_blank>https://mitcc.org/</a>
+ */
+#ifndef MANAGER_API_H
+#define MANAGER_API_H
+
+/*****************************************************************************
+ * INCLUDES
+ *****************************************************************************/
+#include "manager_app.h"
+#include "pb_decode.h"
```

Comment:
include pb_decode and pb_encode only in .c file?

Reply:
updated